lua/nvim-treesitter/highlight.lua

```diff
@@ -53,6 +53,10 @@ hlmap["type.builtin"] = "TSTypeBuiltin"
 hlmap["structure"] = "TSStructure"
 hlmap["include"] = "TSInclude"
 
+-- variable
+hlmap["variable"] = "TSVariable"
+hlmap["variable.builtin"] = "TSVariableBuiltin"
+
```

Comment:
These should be added in CONTRIBUTING.md

Reply:
Fixed @stsewd